santa/src/Santa.Data/Repository/Repository.cs

```diff
@@ -461,7 +461,13 @@ public async Task CreateAssignmentStatus(Logic.Objects.AssignmentStatus newAssig
 
         public async Task<List<Logic.Objects.AssignmentStatus>> GetAllAssignmentStatuses()
         {
-            List<Logic.Objects.AssignmentStatus> listLogicAssigmentStatus = (await santaContext.AssignmentStatus.ToListAsync()).Select(Mapper.MapAssignmentStatus).ToList();
+            List<Logic.Objects.AssignmentStatus> listLogicAssigmentStatus = await santaContext.AssignmentStatus

```

Comment:
Better getter for assignment statuses

Reply:
It just gets data directly from the table, downloading it directly and only the data it needs. Realistically there is no difference, given that it doesnt reach to other tables, but given some of my implimentation is moving towards this kind of architecture, was just namely to match the current implimentation